### ISSUE TITLE
[FEATURE] Improve editor UX and a11y

### ISSUE THREAD

**Issue by dingo-d** [infinum/eightshift-frontend-libs#53]
Currently, the screen for editing is not that user-friendly. 

When you go to the post screen you add title and then nothing happens. Tabbing doesn't do anything. You need to either go down at the red + or in the top corner to add blocks.

The `/` command is also not working, which is a really cool feature of the core editor.

Just using the editor is not simple and doesn't flow well, like in the native editor.

Also, the a11y is not really good. The tab should have a function (new block, select something etc.). We cannot market this as is, because we'll get heavily criticized for lack of a11y support. This is especially important since the entire WP community is going towards standardizing major a11y features (skip links, keyboard navigation, screen reader etc.) in themes and in other areas.

We'll need to discuss how best to fix these issues.

**Comment by iruzevic** [comment]
This can be done but there is possibly an issue in the core. I'm waiting for the answer 

https://github.com/WordPress/gutenberg/issues/15730#issuecomment-757363274

**Comment by iruzevic** [comment]
this is implemented
